src/test/java/be/seeseemelk/mockbukkit/profile/PlayerProfileMockTest.java

```diff
@@ -97,6 +100,22 @@ void setUuid()
 		assertEquals(uuid2, profile.getUniqueId());
 	}
 
+	@Test
+	void setTextures() throws MalformedURLException
+	{
+		UUID uuid = UUID.fromString("22815ad5-2a54-44c0-8f83-f65cfe5310f8");
+		PlayerProfileMock profile = new PlayerProfileMock("Test", uuid);
+		PlayerTexturesMock playerTextures = new PlayerTexturesMock(profile);
+		playerTextures.setSkin(new URL("http://textures.minecraft.net/texture/208bd79e7d56584e57bbeee5f4cb2172d9a56c49ea9df405002efcf213d3ce52"));
+		playerTextures.setCape(new URL("http://textures.minecraft.net/texture/afd553b39358a24edfe3b8a9a939fa5fa4faa4d9a9c3d6af8eafb377fa05c2bb"));
```

Comment:
Do these need to be web links or would a local Picture also work

Reply:
it shouldnt matter much since we dont validate the actual textures.

Reply:
Could we use an easier-to-read URL, then? Make one, say, Google and one GitHub. That would make it easier to see what went wrong if the test fails